doc/topics/cloud/qs.rst

```diff
@@ -0,0 +1,128 @@
+.. _salt-cloud-qs:
+
+=====================
+Salt Cloud Quickstart
+=====================
+
+Salt Cloud is built-in to Salt and is configured on and executed from your Salt
+Master. On some platforms, Salt Cloud is installed by the ``salt-cloud``
+package.
```

Comment:
Only some distros split up the salt-cloud system into different packages. I believe the RHEL-based systems do this, as well as Debian/Ubuntu. However some of the other distros like Arch do not separate them this way.